docs/planning/netstandard-vnext/README.md

```diff
@@ -0,0 +1,116 @@
+# .NET Standard vNext
+
+This document describes the plan for .NET Standard 2.1, which includes the
+definition of its API surface.
+
+## Goals
+
+* **Include `Span<T>`**. In .NET Core 2.1 we've added `Span<T>` which is an
+  array like type that allows representing managed and unmanaged memory in a
+  uniform way and supports cheap slicing. It's at the heart of most performance-
```

Comment:
Nit: move performance- to the next line, as otherwise it displays as "performance- related".